src/ast/passes/codegen_llvm.cpp

```diff
@@ -839,6 +841,11 @@ ScopedExpr CodegenLLVM::visit(Builtin &builtin)
   }
 }
 
+bool CodegenLLVM::shouldForceInitPidNs(const ExpressionList &args)
```

Comment:
Nit: This could just be a free function at the top of this file and not a method.

Reply:
I agree.